hls4ml/converters/onnx_to_hls.py

```diff
@@ -63,6 +63,8 @@ def get_input_shape(graph, node):
     """
     rv = []
     for inp in node.input:
+        if inp == '':
```

Comment:
When is this used? When are you asking for the input node of a node with no inputs?

Reply:
It happens with Resize node, https://onnx.ai/onnx/operators/onnx__Resize.html